.github/CODEOWNERS

```
@@ -498,7 +498,7 @@
 /test/integration/                      @DataDog/container-integrations
 /test/integration/serverless            @DataDog/serverless
 /test/integration/serverless_perf       @DataDog/serverless
-/test/kitchen/                          @DataDog/agent-platform
+/test/kitchen/                          @DataDog/agent-build-and-releases
```

IMO, same as E2E testing, Agent Developer Tools owns the test frameworks, Agent Build and Release owns the tests themselves, which would give:

```suggestion
/test/kitchen/                          @DataDog/agent-developer-tools
/test/kitchen/test-definitions/         @DataDog/agent-build-and-releases
/test/kitchen/test/integration/         @DataDog/agent-build-and-releases

```